charts/datadog/ci/agent-otel-collector-values.yaml

```diff
@@ -0,0 +1,36 @@
+targetSystem: "linux"
+agents:
+  image:
+    repository: datadog/agent-dev
+    tag: nightly-ot-main
+    doNotCheckTag: true
+  containers:
+    agent:
+      env:
+        - name: DD_OTELCOLLECTOR_ENABLED
+          value: "true"
+        - name: DD_HOSTNAME
+          value: "datadog"
+datadog:
+  apiKey: "00000000000000000000000000000000"
+  appKey: "0000000000000000000000000000000000000000"
+  otelCollector:
+    enabled: true
+    config: |
+      receivers:
+        otlp:
+      exporters:
+        datadog:
+          api:
+            key: "00000000000000000000000000000000"
+      service:
+        pipelines:
+          traces:
+            receivers: [otlp]
+            exporters: [datadog]
+          metrics:
+            receivers: [otlp]
+            exporters: [datadog]
+          logs:
+            receivers: [otlp]
+            exporters: [datadog]
```

Comment:
Can we make the default configuration shorter for otel. like only to have to set `datadog.otelcollector.enabled: true` 

When I saw the dash demo, it looks like only 2 parameters was needed.

I hope that we can simplify at least like

```yaml
agents:
  image:
    tagSuffix: ot-beta
datadog:
  apiKey: "00000000000000000000000000000000"
  appKey: "0000000000000000000000000000000000000000"
  otelCollector:
    enabled: true
```

so my questions are, when `datadog.otelcollector.enabled: true` can we:
* generate a default configuration that use the API provide in the `datadog.apiKey` field?
* set `agents.image.tagSuffix` with `ot-beta`?
* inject the needed envvars in the core-agent?

Reply:
This is a good feedback. Thanks for same. I have addressed same in the latest commit . 
if `datadog.otelcollector.enabled: true` we do the following
1.  set agents.image.tagSuffix with ot-beta if image tag isn't set
2. inject the needed envvar `DD_OTELCOLLECTOR_ENABLED` in the core-agent . 
3. if `datadog.otelcollector.config` is not provided, use the default config.